src/stan-users-guide/odes.Rmd

```diff
@@ -270,7 +260,9 @@ system parameters `theta` and initial state `y0` are read in as data
 along with the initial time `t0` and observation times `ts`. The
 generated quantities block is used to solve the ODE for the specified
 times and then add random measurement error, producing observations
-`y_hat`.  Because the system is not stiff, the `rk45` solver is used.
+`y_hat`.  Because the system is not stiff, the `rk45` solver is used
+(in fact, the system being linear, the most efficient approach would
```

Comment:
I am not really an expert on matrix-exponentials... but aren't some matrix-exp solvers using RK45 internally? I am not sure if this statement really holds in that generality. To my understanding it really depends on the properties of the eigensystem of the matrix whatever is best. I guess you want to lead people to the matrix exponential, but the real benefit of that one is to use the matrix exponential action which is right now not written in an efficient way due to memory issues.

In short: Would you mind making the statement less general on matrix exponentials?

Reply:
Fixed.

Stan uses the Pade method coupled with scaling and squaring. My understanding is that the matrix exp method is more efficient because you only compute the solution at the required times, and don't need intermediate times. The exception I've seen involve stiff systems, and my conjecture is that the matrix exp is less efficient because of numerical instability (i.e. dealing with matrix whose elements have widely different orders of magnitude). But I haven't formally tested this conjecture.